libraries/session-storage/src/main/sqldelight/io/element/android/libraries/matrix/session/SessionData.sq

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE SessionData (
+    userId TEXT NOT NULL PRIMARY KEY,
```

Comment:
I wanted to avoid this because I saw no point on doing an extra hash operation to create an id when you already have a valid id for free. I could understand using those for values that the user or some attacker might see such as the data folder, but I'm not really convinced about why it's necessary to have that same overhead for each query.

Reply:
OK, let's keep it like this then, since this table will not be a big table, so using potentially big strings for primary key is not a problem.